src/components/AdminSettings.vue

```diff
@@ -193,6 +193,29 @@
 				</div>
 			</settings-checkbox>
 		</div>
+		<div v-if="isSetup" id="ui-defaults" class="section">
+			<h2>{{ t('richdocuments', 'UI Defaults') }}</h2>
+			<settings-checkbox v-model="settings.uidefaults.enabled" :label="t('richdocuments', 'Override UI Defaults')" hint=""
+				:disabled="updating" @input="updateUIDefaults" />
```

Comment:
I actually don't think we should have individual settings for this and instead pick sane defaults, what do you think @jancborchardt

Reply:
loolwsd.xml will also have these options and this will override them if checked.

Reply:
Agree with @juliushaertl here, we shouldn't expose these as settings and simply set our defaults.